mk/modules.mk

```diff
@@ -160,7 +160,10 @@ HAVE_SPEEXDSP := \
 endif
 ifneq ($(USE_MPG123),)
 ifneq ($(HAVE_SPEEXDSP),)
-USE_MPA  := $(shell [ -f $(SYSROOT)/include/lame/lame.h ] || \
+USE_MPA  := $(shell [ -f $(SYSROOT)/include/twolame.h ] || \
+	[ -f $(SYSROOT)/local/include/twolame.h ] || \
+	[ -f $(SYSROOT_ALT)/include/twolame.h ] || \
+	[ -f $(SYSROOT)/include/lame/lame.h ] || \
```

Comment:
I think this logic should be AND instead of OR.

perhaps add another test e.g. `HAVE_TWOLAME`

this modules depends on many libraries:

- libspeexdsp
- libmpg123
- libmp3lame
- libtwolame

Reply:
Very true.
Hopefully this is better...